src/Feature/Changelog/Feature_Changelog.re

```diff
@@ -128,6 +133,11 @@ module View = {
   module Styles = {
     open Style;
 
+    let typWidth = 60;
```

Comment:
Minor: Should this be `typeWidth` instead of `typWidth`?

Reply:
The name of the component is `typ` (I assume because `type` is a reserved keyword, this was there when I started working), so I thought I would be consistent with that. I can change it if you think it's clearer the other way!

Reply:
Ah, got it! That makes sense to be consistent with the component 👍